File: 231211/알파벳에 따른 평가/evaluation-by-alphabet.js
let fs = require("fs");
let n = fs.readFileSync(0).toString().trim();

if (n === "S") {
    console.log("Superior");
}
else if (n === "A") {
    console.log("Excellent");
}
else if (n === "B") {
    console.log("Good");
}
else if (n === "C") {
    console.log("Usually");
}
else if (n === "D") {
    console.log("Effort");
}
else {
    console.log("Failure");
}

File: 240712/알파벳에 따른 평가/evaluation-by-alphabet.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);
        String s = sc.next();
        char a = s.charAt(0);

        if(a=='S'){
            System.out.print("Superior");
        }
        else if(a=='A'){
            System.out.print("Excellent");
        }
        else if(a=='B'){
            System.out.print("Good");
        }
        else if(a=='C'){
            System.out.print("Usually");
        }
        else if(a=='D'){
            System.out.print("Effort");
        }
        else{
            System.out.print("Failure");
        }
    }
}